docs/releases/v2019.1.rst

```diff
@@ -0,0 +1,233 @@
+Gluon 2019.1
+############
+
+Important notes
+***************
+
+If you are upgrading from a version prior to v2018.1 please note that the flash layout on some 
+devices (TP-Link CPE/WBS 210/510) was changed. To avoid upgrade failures, make sure to upgrade
+to Gluon 2017.1.8 or the latest Gluon 2016.2.x (unreleased) before installing Gluon 2018.1, 2018.2 or 2019.1.
+
+Routers will now only respond to queries on ``[ff05::2]:1001``, this will most likely required
+updates to your respondd queriers. This change was required due to cross-domain leakage of respondd data.
+
+Added hardware support
+**********************
+
+ar71xx-generic
+==============
+
+* D-Link
+
+  - DAP-1330 A1
+
+* TP-Link
+
+  - CPE210 v3
+
+
+ar71xx-nand
+===========
+
+* Aerohive
+
+  - HiveAP 121
+
+ar71xx-tiny [#deprecated]_
+==========================
+
+* TP-Link
+
+  - TL-WR840N v2
+
+ipq40xx
+=======
+
+* 8devices
+
+  - Jalapeno
+
+mpc85xx-p1020 [#newtarget]_
```

Comment:
target `mpc85xx-p1020` with `Aerohive HiveAP 330` are part of v2018.2.3 already

https://github.com/freifunk-gluon/gluon/blob/v2018.2.x/targets/mpc85xx-p1020

Reply:
Agreed, but communities might move from v2018.2.2 to v2019.1 and otherwise miss the new target.